[View in Colaboratory](https://colab.research.google.com/github/Bhatnagar-Ashish/test/blob/master/Assignment2B.ipynb)

In [22]:
import numpy as np

weight = np.around(np.random.random((4,3)),decimals=2)  # Create an 4X3 array filled with random values
print("Weight: ")
print(weight) 

bias=np.around(np.random.random((1,3)),decimals=2)  # Create an 1X3 array filled with random values
print("Bias: ")
print(bias) 

Weight: 
[[0.17 0.61 0.61]
 [0.28 0.18 0.26]
 [0.23 0.82 0.32]
 [0.83 0.58 0.77]]
Bias: 
[[0.68 0.64 0.73]]


### Weight 

$\mathbf{Wh} = \left[\begin{array}
{rrr}
0.17  & 0.61 & 0.61 \\
0.28 & 0.18 & 0.26 \\
0.23 & 0.82 & 0.32 \\
0.83 & 0.58 & 0.77
\end{array}\right]
$


### Bias

$\mathbf{Bh} = \left[\begin{array}
{rrr}
0.68 & 0.64 & 0.73
\end{array}\right]
$